src/sql/azdata.proposed.d.ts

```diff
@@ -444,6 +444,23 @@ declare module 'azdata' {
 		errorMessage?: string;
 	}
 
+	export interface ProviderError {
```

Comment:
So now that prompt can return this as well you should be updating the return type to include this - although having two separate types of "errors" is definitely a bit weird...(this and PromptFailedResult).

Looking at PromptFailedResult, why aren't you just using that anyways? The only new thing you're adding here is `name` so is there a reason you're not just adding this to the PromptFailedResult?

Reply:
I wanted a more generic error, but think I can just change the name of PromptFailedResult